### ISSUE TITLE
SKR3 EZ Sensorless Homing not working

### ISSUE THREAD

**Issue by asdino2** [bigtreetech/SKR-3#64]
motors don't stop at the end of bed. still running . Jumpers are on the board . If I connect the motor and give direction , will run 200, 300 400 mm even the bed is 200mm

**Comment by radek8** [comment]
Did you compile your own FW?
How big did you make the bed?
Did you enable SW endstop?

**Comment by asdino2** [comment]
Yes , I compiled on Marlin 2.1.2.1 . Bed is 245x 205 for a Prusa mk3 . Yes software end stop are enable . I put he jumper also . Your EZ 2209 seems not to see the overcurrent . I set up for 800 mA and nothing . Changed with 200 mA or 1000mA and still nothing Already I lost 1 EZ 2209 module . 
 
 
  On Tue, Jun 6, 2023 at 2:34 p.m., ***@***.***> wrote:   


Did you compile your own FW?
How big did you make the bed?
Did you enable SW endstop?

**Comment by radek8** [comment]
Do you use sensorless guidance?

**Comment by asdino2** [comment]
What is that ? I use the settings  from the SKR3EZ manual
 
 
  On Tue, Jun 6, 2023 at 9:44 p.m., ***@***.***> wrote:   


Do you use sensorless guidance?

**Comment by radek8** [comment]
If you execute the command home (G28)
does the print head stop on the microswitch or are you using sensorless guidance and the head hits the frame?

**Comment by asdino2** [comment]
The X or y hits the frame 
 
 
  On Wed, Jun 7, 2023 at 9:49 a.m., ***@***.***> wrote:   


If you execute the command home (G28)
does the print head stop on the microswitch or are you using sensorless guidance and the head hits the frame?

**Comment by radek8** [comment]
I see
I didn't notice the title
Sensorless Homing
So you are using sensorless guidance.
you have to fit the jumpers.

![image](https://github.com/bigtreetech/SKR-3/assets/46979052/2cdc7e83-e6f4-4c19-a3d1-be6167206f21)

You need to enable guidance in Marlin and set the parameter for your engines

#define SENSORLESS_HOMING 
#define X_STALL_SENSITIVITY
#define Y_STALL_SENSITIVITY

**Comment by asdino2** [comment]
I did already that , as I explained,  and is not working,  for this reason I write you . I follow  the manual . I put Mac sensitivity 250 and nothing. 
Thanks
 
 
  On Wed, Jun 7, 2023 at 11:15 a.m., ***@***.***> wrote:   


I see
I didn't notice the title
Sensorless Homing
So you are using sensorless guidance.
you have to fit the jumpers.



You need to enable guidance in Marlin and set the parameter for your engines

#define SENSORLESS_HOMING
#define X_STALL_SENSITIVITY
#define Y_STALL_SENSITIVITY

**Comment by radek8** [comment]
By changing the value
#define X_STALL_SENSITIVITY
#define Y_STALL_SENSITIVITY
you change the sensitivity. you need to find the correct value for your printer

**Comment by asdino2** [comment]
ok, I will try every number from 0 to 255.I have 1 EZ2209 which I insert inside is overheating the chip and is not working. Can you replaced in warranty ?I have also ESP32U which in manual is write it is possible to flash the firmware buy the microSD. Is not possible , I tried and is not communicate with the motherboard 
Thank you  
    On Wednesday, June 7, 2023 at 11:25:26 a.m. MDT, Radek ***@***.***> wrote:  
 
 


By changing the value
#define X_STALL_SENSITIVITY
#define Y_STALL_SENSITIVITY
you change the sensitivity. you need to find the correct value for your printer

**Comment by radek8** [comment]
I'm not from BTT. 
BTT does not answer here. 
You must contact the dealer to claim a defective part.

**Comment by radek8** [comment]
Yes, you need to find the correct value between 0 and 255.
If the value is low or high, the engine will not spin or hit the frame..
I use microswitches. This feature behaved unreliably for me

**Comment by asdino2** [comment]
Thank you for help me , I tried all the numbers from 1 to 255 and no end  stopping . Sensorless homing not working on skr3 ez for sure . I ask also other blogers on you tube and they did not test this function . they use end stop switch .what else I can do ? Is not working 
    On Wednesday, June 7, 2023 at 10:47:30 p.m. MDT, Radek ***@***.***> wrote:  
 
 


Yes, you need to find the correct value between 0 and 255.
If the value is low or high, the engine will not spin or hit the frame..
I use microswitches. This feature behaved unreliably for me

**Comment by radek8** [comment]
Do you have a diag jumper fitted?

**Comment by radek8** [comment]
How are you on the impact sensitivity values?
Did you compile the FW with the new value, or did you use the M914 code?

https://marlinfw.org/docs/gcode/M914.html

**Comment by asdino2** [comment]
I use the m914 code from the screen and save it . After that tried and no response . I am sure the board don't  see the curent sensitivity 
 
 
  On Wed, Jun 7, 2023 at 11:37 p.m., ***@***.***> wrote:   


How are you on the impact sensitivity values?
Did you compile the FW with the new value or did you use the M914 code?

**Comment by radek8** [comment]
Do you have a diag jumper fitted?

**Comment by asdino2** [comment]
Yes, I have the jumper. I forget to tell you sometimes I get a error like "All low " when I power on the board, but not all the time , can be this a problem ? Do I need to set up auto home automatically? 
 
 
  On Thu, Jun 8, 2023 at 7:44 a.m., ***@***.***> wrote:   


Do you have a diag jumper fitted?

**Comment by radek8** [comment]
The "All low" message is a problem.
It seems that some TMCs have a problem with serial communication with the processor.
Send the M122 command and copy the response here.

Send configuration files Configuration.h and configuration_adv.h

What TMC do you have installed? Can you send a photo?

**Comment by asdino2** [comment]
Thank you
    On Thursday, June 8, 2023 at 08:55:55 a.m. MDT, Radek ***@***.***> wrote:  
 
 


The "All low" message is a problem.
It seems that some TMCs have a problem with serial communication with the processor.
Send the M122 command and copy the response here.

Send configuration files Configuration.h and configuration_adv.h

**Comment by ReefWarrior3D** [comment]
Any update on this?

**Comment by asdino2** [comment]
Still not working !!!
 
 
  On Sun, Aug 6, 2023 at 11:01 p.m., ***@***.***> wrote:   


Any update on this?

**Comment by mateo928** [comment]
> Il messaggio "All low" è un problema. Sembra che alcuni TMC abbiano un problema con la comunicazione seriale con il processore. Invia il comando M122 e copia la risposta qui.
> 
> Invia i file di configurazione Configuration.h e configuration_adv.h
> 
> Quale TMC hai installato? Puoi inviare una foto?

hi raden8, can you help me? i have the same problem
